python/bin/xpy_enumerate

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import sys
+from ctypes import POINTER, c_uint32, cast, pointer
+
+import xnvme.ctypes_mapping as capi
+
+capi.guard_unloadable()
```

Comment:
Same here

Reply:
How about this would go on the list of things to refine in ``docs/python/index.rst``?